source/network-security.txt

```diff
@@ -6,40 +6,46 @@ Network Security
 
 .. default-domain:: mongodb
 
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: atlas architecture center
+   :description: Learn about the network security configurations that Atlas supports.
+
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: onecol
 
-{+service+} provides secure defaults for your database deployments
-such as:
+{+service+} provides secure network configuration defaults for your
+database deployments, such as:
 
 - Mandatory |tls-ssl| connection encryption
 - {+vpc+}\s for all projects with one-or-more {+Dedicated-clusters+}
-- Access-list-based authentication which only accepts connections from
-  sources you explicitly declare
+- Authentication which uses {+ip-access-list+}s and only accepts connections
```

Comment:
I think this should be "that" instead of "which" in this phrasing
```suggestion
- Authentication that uses {+ip-access-list+}s and only accepts connections
```